preupg/cli.py

```diff
@@ -54,126 +54,166 @@ def __init__(self, args=None):
+                 " this option. When this option is used in concert with"
+                 " --verbose option, summary of the risks are printed to"
+                 " STDOUT. If the --verbose option is used once, just HIGH and"
+                 " EXTREME risks are printed. If it is used twice, all the"
+                 " risks are printed. " + " " * 30 +
+                 "Return values:" + " " * 40 +
+                 "0 ... SLIGHT/MEDIUM risk or needs_inspection/fixed/"
+                 "informational/not_applicable/not_selected/not_checked/pass"
+                 " result" + " " * 50 +
+                 "1 ... HIGH risk or needs_action result" + " " * 25 +
+                 "2 ... EXTREME risk or error/fail result"
+        )
+        self.parser.add_option(
+            "-f", "--force",
             action="store_true",
             default=False,
-            help="Suppress user interaction"
+            help="Suppress user interaction."
         )
         self.parser.add_option(
-            "--text",
+            "-t", "--text",
             action="store_true",
             default=False,
-            help="Convert HTML results to text form by elinks, lynx or w3m"
+            help="Generate plain text assessment report alongside XML and HTML"
+                 " reports. The text report is converted from HTML using"
+                 " elinks, lynx or w3m tool."
         )
         self.parser.add_option(
-            "-v",
-            "--verbose",
+            "-v", "--verbose",
             action="count",
             default=0,
-            help="Verbose mode"
+            help="Show more information during the assessment."
         )
         self.parser.add_option(
-            "--cleanup",
+            "-C", "--cleanup",
             action="store_true",
             default=False,
-            help="Clean data created by preupgrade-assistant"
+            help="Remove all the files created by previous runs of Preupgrade"
+                 " Assistant."
         )
         self.parser.add_option(
-            "-m",
-            "--mode",
+            "-m", "--mode",
             metavar="MODE",
             choices=['migrate', 'upgrade'],
-            help="Select mode which can be used for migration or upgrade"
+            help="Select what you plan to do with the system after performing"
+                 " its assessment by Preupgrate Assistant - migration or"
+                 " upgrade. Both modes are selected by default. This option"
+                 " may only affect behaviour of the modules - they can provide"
+                 " different results when only one mode is selected. Use one"
+                 " of these values: migrate, upgrade. It may be that modules"
+                 " behave the same no matter what mode is selected."
         )
         self.parser.add_option(
-            "--kickstart",
+            "-k", "--kickstart",
```

Comment:
The option will be probably removed in future. It should be already deprecated - I am not sure why the message is not there now. As the creation of a kickstart file is provided by the separate tool. Rather do not use "-k" here.